xml_converter/generators/code_generator.py

```diff
@@ -353,6 +353,45 @@ def generate_cpp_variable_data(
 
         return attribute_variables, cpp_include_paths
     
+    ############################################################################
+    # write_attributes
+    #
+    # Creates the attribute files that contain multiple values
+    # 
+    ############################################################################
+    def write_attribute (self, output_directory: str) -> None:
+        print("Writing attributes")  
+        os.makedirs(output_directory, exist_ok=True)
+
+        file_loader = FileSystemLoader('cpp_templates')
+        env = Environment(loader=file_loader)
+        template = env.get_template("attribute_template.hpp")
+        categories: Dict[str,List[str]] = {}
+        attribute_names: Dict[str,str] = {}
+        metadata: Dict[str, SchemaType] = {}
+         
+        
+        for filepath in self.data.keys():
+            filename = os.path.basename(filepath)
+            attribute_names[filepath]= filename.split(".md")[0]
+
+        for filepath in attribute_names:
+            attribute_name = attribute_names[filepath]
+            metadata[filepath] = self.data[filepath].metadata
+            if metadata[filepath]['type'] in ["MultiflagValue"]:
```

Comment:
Is this if statement going to be used for all other attributes? Or will there be different code to run for other generated attribute types?

If it is just a single type then `in` does not need to be used, just `==`